Changed the duration of no tags available message

oscarli9 opened the pull request:
Added two other configurations for styles based on Crouton text duration, changed the duration of no tags available message from DURATION_SHORT(950ms) to DURATION_MEDIUM(1650ms) to fix issue #663

federicoiosue commented:
Hi Oscar, in which way this would fix the issue? Isn't that a matter of ratio of text and delimited space sizes?

oscarli9 commented:
Hi Federico. In the additional context of Issue #663, Adeel mentioned that "it's difficult to read all of the text in the message within the duration as it is very less" so I added more configurational options for message text duration and changed the original duration. For the issue mentioned in the title - text cut, I tried to reproduce the issue across many types of emulators. I found text cut would not happen on most of them but only on Galaxy Nexus as Adeel said, so I didn't really change the original text size/space.
